packages/aws-lambda-otel-extension/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.2.21](https://github.com/serverless/runtime/compare/@serverless/aws-lambda-otel-extension@0.2.19...@serverless/aws-lambda-otel-extension@0.2.21) (2022-04-26)
+
+### Bug Fixes
+
+- Update timeout message ([3596dca](https://github.com/serverless/runtime/commit/3596dca7b927958b7e6e4addda646bc8820b8da7))
```

Comment:
Before merge we don't know the commit sha (on `main` branch this commit will have different hash), so to have valid changelogs we need to first merge the feature, and then create release PR (which is just about changelog and version bump)

Also before creating release it's good to confirm that integration tests pass (although I agree that in case of this PR it's hard to get it wrong).